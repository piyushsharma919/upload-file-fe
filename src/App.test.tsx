import { render, screen } from '@testing-library/react'
import App from './App.tsx'

describe('App', () => {
  it('renders the App component', () => {
    render(<App />)

    screen.debug();
  })

  it('renders without crashing', () => {
    render(<App />)
    expect(screen.getByText('Hello World!!!!')).toBeInTheDocument()
  })

  it('matches snapshot', () => {
    const { asFragment } = render(<App />)
    expect(asFragment()).toMatchSnapshot()
  })
})
