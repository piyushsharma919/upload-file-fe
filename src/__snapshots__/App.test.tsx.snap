// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h1>
      Hello World!!!!
    </h1>
  </div>
</DocumentFragment>
`;
